css/main.css

```diff
@@ -0,0 +1,51 @@
+@charset "UTF-8";
+
+@font-face {
+   font-family: "CrystalUniverse-Oblique.ttf";
```

Comment:
you don't need .ttf in the family name. Likewise for the other font family. It's a detail about the original file. You'll also want to run the file through https://transfonter.org/ to get more file formats for browser compatibility